### PR TITLE
BUGFIX: Security Remap

### PR DESCRIPTION
**Pull request by Eyeveri**
Fixed a small pressure issue from missing vent and scrubber. 
Fixed a floating fire alarm and missing lightbulb.
Fixed a missing maintenance door.

### CODE REVIEW

**Comment by Eyeveri** [comment]
!bugfix

**Comment by Eyeveri** [comment]
!review

File: html/changelogs/Eyeveri_Bugfix.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.
+author: ReadThisNamePlz
```
Comment (Arrow768) [inline html/changelogs/Eyeveri_Bugfix.yml:30]: Changelog 1

File: html/changelogs/Eyevery_SecurityRemap.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.
+author: ReadThisNamePlz
```
Comment (Arrow768) [inline html/changelogs/Eyevery_SecurityRemap.yml:30]: Changelog 2

**Review by Arrow768 — CHANGES_REQUESTED** [review]
Duplicate changelog

File: html/changelogs/Eyevery_SecurityRemap.yml (hunk 2 above)
Comment (Eyeveri) [reply html/changelogs/Eyevery_SecurityRemap.yml:30]: will remove the first one

**Review by Arrow768 — APPROVED** [review]

**Comment by ScheveningenZonn** [comment]
image

this tile doesn't have wiring across it to connect to the armory APC

**Comment by Eyeveri** [comment]
> image
> 
> this tile doesn't have wiring across it to connect to the armory APC

I know, I fixed it with this merge

**Pull request merged**